application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Macro.xml

```diff
@@ -1201,7 +1202,7 @@ require(['jquery', 'moccaCalendar'], function(jQuery) {
     dayNamesShort: $services.localization.render('xwiki.calendar.dayNamesShort'),
     allDayText: "$escapetool.javascript($services.localization.render('xwiki.calendar.allDayText'))",
     axisFormat: "$escapetool.javascript($services.localization.render('xwiki.calendar.axisFormat'))",
-    timeFormat: "$!escapetool.javascript($services.localization.render('xwiki.calendar.timeFormat'))",
+    timeFormat: "$dateFormat.substring($dateFormat.indexOf('H'))",
```

Comment:
A few comments here:

* when we stop using a translation key we need to check if it's used elsewhere (inside the module) and if not then mark it as deprecated in the translation page/file
* what happens if the date format doesn't include a time format? Since the date format can be customized the user could set it to ``dd.MM.yyyy`` to only show the date and not the time
* AFAIK the hour format can be specified in [multiple ways](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html) (H, h, K, k with different meanings); I assume we need to take this into account.
* there's no imposed order between date an time within a date format; you can display the time before the date if you want, so this way of extracting the time format from the date format is very fragile

So if we want to extract the time format from the date format then we need to:
* have a fallback, in case the time format is not specified
* document that we expect from the users to specify the time format after the date format, otherwise the default time format is used
* support all hour formats (don't look only for ``H``)